config/samples/deployment.yaml

```
@@ -129,7 +129,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: foundationdb/fdb-kubernetes-operator:v0.49.0
+        image: foundationdb/fdb-kubernetes-operator:v0.49.2
```

```suggestion
        image: foundationdb/fdb-kubernetes-operator:v0.50.0
```

We already released the `v0.50.0` version so we want to update to the latest version here. I guess we missed that step in the release process.